formbuilder/config.js

```diff
@@ -72,6 +72,39 @@ export default {
       formData: {}
     },
     {
+      id: "multiple-checkbox",
+      icon: "check",
+      label: "Multiple choices",
+      jsonSchema: {
+        type: "array",
+        title: "A multiple choices list",
+        items: {
+          type: "string",
+          enum: ["choice 1", "choice 2", "choice 3"],
+        },
+        uniqueItems: true,
+      },
+      uiSchema: {
+        "ui:widget": "checkboxes",
+        editSchema: {
+          type: "object",
+          properties: {
+            title: {type: "string", title: "Label"},
+            required: {type: "boolean"},
+            enum: {
+              type: "array",
+              title: "Choices",
+              items: {
+                type: "string"
+              },
+              default: ["choice 1", "choice 2", "choice 3"],
+            }
+          }
+        },
+      },
+      formData: {}
```

Comment:
And of course when you read it it all looks so simple. If only I hadn't tried :-)